google/generativeai/responder.py

```diff
@@ -0,0 +1,467 @@
+# -*- coding: utf-8 -*-
```

Comment:
Wasn't this already created in https://github.com/google-gemini/generative-ai-python/pull/289 ?

Maybe you need to merge or rebase?